stylesheets/_component.navigation.scss

```diff
@@ -360,6 +366,115 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
     }
 }
 
+.nav-tertiary {
```

Comment:
There's a lot of duplication here with `nav-tertiary` and `nav-quaternary`, could this be DRYed out somewhat?